addon/serializers/active-model-serializer.js

```diff
@@ -38,6 +38,11 @@ export default Serializer.extend({
   normalize(payload) {
     let type = Object.keys(payload)[0];
     let attrs = payload[type];
+    let modelName = camelize(type);
+    let model = this.schema.modelFor(modelName);
+    let { belongsToAssociations, hasManyAssociations } = model.class.prototype;
```

Comment:
You can get these from [schema.modelClassFor(modelName)](https://github.com/samselikoff/ember-cli-mirage/blob/master/addon/orm/schema.js#L214). (There's also `schema.associationsFor(modelName)` but since you need the hasMany and belongsTo split I think modelClassFor is the best approach).